Cartfile

```diff
@@ -1,3 +1,3 @@
-github "kirualex/SwiftyGif" ~> 5.0
+github "kirualex/SwiftyGif" ~> 5.4.1
```

Comment:
Are we sure we want to depend on specific versions? This will requires to make release everytime there is a crash fixed on our dependencies

Reply:
With this we're saying -> use `5.4.1` or above, up to the next major.

You can read more about the version requirements (here)[https://github.com/Carthage/Carthage/blob/master/Documentation/Artifacts.md#version-requirement].

Reply:
On cocoapods this is actually up to next minor